MelteCompiler.js

```diff
@@ -344,12 +359,10 @@ SvelteCompiler = class SvelteCompiler extends CachingCompiler {
             : { code: processedCode };
         },
         style: async ({ content, attributes }) => {
-          if (this.postcss) {
-            if (attributes.lang == 'postcss') {
-              return {
-                code: await this.postcss.process(content, { from: undefined })
-              };
-            }
+          if (this.postcss && (!attributes.lang || attributes.lang === 'postcss')) {
```

Comment:
In my fork I also introduced a change to the config logic, so that PostCSS is used if it's configured and no other lang-attribute is present.

Reply:
Could you please remove this change from this PR since it is in #6 ? It will require increasing the major version which I am not ready to do yet.

Reply:
I reverted this back to previous behaviour.